scripts/src/public/services/orders.ts

```diff
@@ -78,6 +78,15 @@ export async function getOrder(orderId: string, logger: LoggerInstance): Promise
 	return orderDbToApi(order);
 }
 
+export async function getOpenOrder(orderId: string): Promise<db.MarketplaceOrder| db.ExternalOrder> {
+	const order = await db.Order.getOne(orderId, "opened") as db.MarketplaceOrder | db.ExternalOrder;
+
+	if (!order) {
```

Comment:
why?
won't it be better to return `Promise<db.MarketplaceOrder| db.ExternalOrder | undefined>` and then whoever called this function will deal with it?